### PR TITLE
gmtk 1.4.4 (new formula)

### PR DESCRIPTION
**Pull request by rprogers**
This adds the [Graphical Models Toolkit (GMTK)](http://melodi.ee.washington.edu/gmtk/) - a dynamic graphical model prototyping system used by the computational biology & speech recognition research communities and the DARPA [TerraSwarm project](http://www.terraswarm.org/).

### CODE REVIEW

File: Library/Formula/gmtk.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (bfontaine) [inline Library/Formula/gmtk.rb]: Please remove this empty line.

File: Library/Formula/gmtk.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--without-x",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "gmtkJT", "-version"
```
Comment (bfontaine) [inline Library/Formula/gmtk.rb]: Use `"#{bin}/gmtkJT"` instead.

File: Library/Formula/gmtk.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--without-x",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"smoke_test.str").write("GRAPHICAL_MODEL smoke_test frame: 0 { variable: X { type: discrete observed 0:0 cardinality 2; conditionalparents: nil using DenseCPT(\"X0\"); } } chunk 0:0")
```
Comment (DomT4) [inline Library/Formula/gmtk.rb]: Can we shrink this line at all, or extend it into a block? It's currently making the formula quite lengthy. We try to keep things as close to the 80 character width as possible for readability.

**Comment by bfontaine** [comment]
This is probably a good fit for out [`homebrew/science`](https://github.com/Homebrew/homebrew-science) tap.

**Comment by bfontaine** [comment]
@rprogers The formula looks good; but please submit it to [`homebrew/science`](https://github.com/Homebrew/homebrew-science) instead. Thanks!

**Pull request closed without merging**